app/models/trip.rb

```diff
@@ -0,0 +1,36 @@
+class Trip < ApplicationRecord
+  has_many :trip_weathers
+  has_many :weathers, through: :trip_weathers
+
+  has_many :request_trips
+  has_many :requests, through: :request_trips
+
+  def self.new_trip(flight, weather, index)
```

Comment:
Good work here.

Reply:
Looks like this effectively replaces the `initialize` method in our `Trip` poro. Nice!

Reply:
Awesome, I didn't know if we wanted to delete these files if they are no longer being used.